cds_ils/circulation/serializers/json.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
```

Comment:
This class is probably not needed

Reply:
The json.py serializer was created because originally the process of deleting unnecessary fields was also used on the JSONs, as the algorithm I wrote could be easily reused there. In the implementation in invenio-records-rest, there is only the option of ignoring fields in the csv serializer. Originally the objective was only to delete fields on the csv serializer so, in the end, the json.py class is truly not needed.